tests/new_excerpt.py

```diff
@@ -0,0 +1,30 @@
+import pytest
+from selenium import webdriver
+from selenium.webdriver.common.by import By
+
+
+def new_excerpt(selenium, base_url):
```

Comment:
Consider calling the first argument `driver`, or `webdriver` or `selenium_webdriver` or similar, as it isn't the selenium package. That the test functions which call this function obtain the value they will pass here from the pytest fixture also called `selenium` shouldn't bother this function here.